plone/scale/scale.py

```diff
@@ -494,6 +552,41 @@ def scalePILImage(image, width=None, height=None, mode="scale", direction=None):
         # scaling, so return the potentially pre cropped image
         return image
 
+    # Try using OpenCV for better performance if it's available
+    if HAS_OPENCV:
```

Comment:
Why does this duplicate the code from _scale_with_opencv instead of calling it?

Reply:
Ah, Sorry. Thanks @davisagli for pointing out... maybe I was half asleep...